st2common/st2common/router.py

```diff
@@ -71,8 +71,6 @@ def op_resolver(op_id):
     method_callable = functools.reduce(getattr, func_name.split('.'), module)
 
     return controller_instance, method_callable
-    print('xxx')
-    print(controller_instance)
     return functools.reduce(getattr, func_name.split('.'), module)
```

Comment:
None of this code after line 74 is reachable, it might be good to simply remove the second `return`.

Reply:
Will remove it, looks like this "debug" code has been there for a while, lol.